pkg/apis/provisioning/v1alpha5/suite_test.go

```diff
@@ -185,13 +185,6 @@ var _ = Describe("Validation", func() {
 				Expect(provisioner.Validate(ctx)).To(Succeed())
 			}
 		})
-		It("should fail because no feasible value", func() {
```

Comment:
This is no longer enforced at the API level, and is caught during compatibility checks.